distribution/docs/src/main/resources/_contents/quickstartcontents.adoc

```diff
@@ -0,0 +1,122 @@
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+
+These steps will demonstrate:
+
+- [*] Prerequisites
+- [*] Installation
+- [*] Ingesting into the ${ddf-branding} Catalog
+
+==== Prerequisites
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
+* http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
+* The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+
+.Setting `JAVA_HOME` on *NIX
+----
+JAVA_HOME=/usr/java/jdk1.8.0
```

Comment:
these usually include an appended "_[buildnumber]" do we note that here or trust our users to be smart enough to figure it out?